src/main/java/com/rarchives/ripme/ripper/rippers/RedditRipper.java

```diff
@@ -59,6 +63,14 @@ private URL getJsonURL(URL url) throws MalformedURLException {
 
     @Override
     public void rip() throws IOException {
+
+        // once we begin the rip process, clear download.rip_authors
+        // the value is preserved for this session and will not
+        // persist when getAndParseAndReturnNext spawns additional
+        // AbstractRipper processes
+        Utils.setConfigBoolean("download.rip_authors", false);
```

Comment:
Couldn't this configBoolean also come from the `rip.properties` file? Maybe we should have options that don't coordinate with the `rip.properties` so we don't need to hack around it and clear it, even if the user wanted to specify it in the config to always be used.